Regression: PTF docker image does not support ptf_nn_agent with pynng

### Is it platform specific

generic

### Importance or Severity

Medium

### Previous Working Version

previous docker-ptf version

### Steps to Reproduce

ptf_nn_agent cannot start due to

```
root@50159ae68f06:~# /usr/bin/python /opt/ptf_nn_agent.py --device-socket 0@tcp://127.0.0.1:10900 -i 0-3@eth3
Cannot find pynng package, please ins
```

This is because of https://github.com/p4lang/ptf/pull/234 drops support for nnpy.


### Impact of this regression

ptf related container will fail

### Relevant log output

```shell
if (hostname_res.is_failed or 'exception' in hostname_res) and not module_ignore_errors:
>           raise RunAnsibleModuleFail("run module {} failed".format(module_name), hostname_res)
E           tests.common.errors.RunAnsibleModuleFail: run module command failed, Ansible Results =>
E           failed = True
E           changed = True
E           rc = 7
E           cmd = ['supervisorctl', 'restart', 'ptf_nn_agent']
E           start = 2026-04-21 06:06:35.673222
E           end = 2026-04-21 06:06:36.799536
E           delta = 0:00:01.126314
E           msg = non-zero return code
E           invocation = {'module_args': {'_raw_params': 'supervisorctl restart ptf_nn_agent', '_uses_shell': False, 'expand_argument_vars': True, 'stdin_add_newline': True, 'strip_empty_ends': True, 'argv': None, 'chdir': None, 'executable': None, 'creates': None, 'removes': None, 'stdin': None}}
E           _ansible_no_log = False
E           stdout =
E           ptf_nn_agent: stopped
E           ptf_nn_agent: ERROR (spawn error)stderr =
```

### Output of `show version`, `show techsupport`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/copp/copp_utils.py
+++ b/tests/copp/copp_utils.py
@@ -228,7 +228,7 @@ def _install_nano_bookworm(dut, creds, syncd_docker_name):
                 && apt-get update \
                 && apt-get install -y python3-pip build-essential libssl-dev libffi-dev \
                 python3-dev python3-setuptools wget libnanomsg-dev python-is-python3 \
-                && TMPDIR=/var/tmp_build pip3 install --no-cache-dir cffi==1.16.0 \
+                && TMPDIR=/var/tmp_build pip3 install --no-cache-dir cffi \
                 && TMPDIR=/var/tmp_build pip3 install --no-cache-dir nnpy \
                 && rm -rf /var/tmp_build \
                 && mkdir -p /opt && cd /opt && wget {3} \
@@ -249,8 +249,9 @@ def _install_nano(dut, creds,  syncd_docker_name):
             creds (dict): Credential information according to the dut inventory
     """
 
-    if "bookworm" in dut.shell("docker exec {} grep VERSION_CODENAME /etc/os-release"
-                               .format(syncd_docker_name))['stdout'].lower():
+    codename = dut.shell("docker exec {} grep VERSION_CODENAME /etc/os-release"
+                         .format(syncd_docker_name))['stdout'].lower()
+    if "bookworm" in codename or "trixie" in codename:
         _install_nano_bookworm(dut, creds, syncd_docker_name)
         return
 
@@ -543,7 +544,7 @@ def is_trap_installed(duthost, trap_id, namespace=None):
     """
 
     output = parse_show_copp_configuration(duthost, namespace)
-    assert trap_id in output, f"Trap {trap_id} not found in the configuration"
+    assert trap_id in output, "Trap {} not found in the configuration".format(trap_id)
     assert "hw_status" in output[trap_id], f"hw_status not found for trap {trap_id}"
 
     return output[trap_id]["hw_status"] == "installed"

